addon/components/base/bs-modal/dialog.js

```diff
@@ -134,6 +136,44 @@ export default Component.extend({
     return isBlank(size) ? null : `modal-${size}`;
   }).readOnly(),
 
+
+   /**
+   * The id of the `.modal-title` element
+   *
+   * @property titleId
+   * @type string
+   * @default null
+   * @private
+   */
+  titleId: null,
+
+/**
+   * Gets or sets the id of the title element for aria accessibility tags
+   *
+   * @method getSetTitleID
+   * @private
+   */
+  getOrSetTitleId() {
+      //Title element may be set by user so we have to try and find it to set the id
+      const modalNode = document.getElementById(this.get('id'));
+      let nodeId = null;
+
+      if (modalNode) {
+        const titleNode = modalNode.getElementsByClassName('modal-title');
```

Comment:
Would prefer to use `modalNode.querySelector('.modal-title')` here. This already returns a single DOM node, so no need for `[0]`.